devdoc/CodeSubmissions.md

```diff
@@ -6,7 +6,19 @@ We welcome submissions to the platform especially in the form of pull requests i
 There are a few things that should be considered when creating a PR to increase the likelihood that they can be accepted quickly.
 
 * Write a good commit message\
-  The format should be HPCC-XXXXX (where XXXXX is the bug number) followed by a description of the issue.  The text should make sense in a change log by itself - without reference to the jira or the contents of the PR.  We should aim to increase the information that is included as part of the commit message - not rely on on the jira.
+  The format should be HPCC-XXXXX (where XXXXX is the bug number) followed by a description of the issue.  The text should make sense in a change log by itself - without reference to the jira or the contents of the PR.  We should aim to increase the information that is included as part of the commit message - not rely on the jira.
```

Comment:
is there a maximum length for the commit message?

"makes sense in a change log" - should you mention that the target audience is not developers, but ops, qa, as well as developers

Reply:
@ghalliday I mention later in one of the examples that the character limit is 80.  Is that a true statement or is it 60?